docs/integrations/prefect-dbt/index.md

```diff
@@ -277,26 +278,10 @@ DbtCoreOperation.load("DBT-CORE-OPERATION-BLOCK-NAME-PLACEHOLDER")
 
 ### Feedback
```

Comment:
@discdiver I think we should also remove this section and the contributing section now that this is in the core repo

Reply:
Good call. Done.